src/Magnum/ImGuiIntegration/Test/ContextGLTest.cpp

```diff
@@ -114,8 +114,11 @@ struct Application {
         None = 999
     };
 
-    void setCursor(Cursor cursor) { currentCursor = cursor; }
-    Cursor currentCursor = Cursor::None;
+    Cursor _currentCursor = Cursor::None;
+    Vector2i _mousePos;
```

Comment:
```suggestion
    Cursor currentCursor = Cursor::None;
    Vector2i mousePos;
```

This could have stayed non-underscored, they're public variables.